ast/term.go

```diff
@@ -1592,6 +1592,7 @@ type Object interface {
 	MergeWith(other Object, conflictResolver func(v1, v2 *Term) (*Term, bool)) (Object, bool)
 	Filter(filter Object) (Object, error)
 	Keys() []*Term
+	Elem(i int) (*Term, *Term)
```

Comment:
I'd probably add a private function to the interface to prevent external implementations at compile-time.

Reply:
Sure, added get().